### PR TITLE
RavenDB-12332 Show cert renewal date

### PR DESCRIPTION
**Pull request by ml054**

### CODE REVIEW

File: src/Raven.Studio/typescript/viewmodels/manage/certificates.ts [hunk 1]
```diff
@@ -178,15 +184,30 @@ class certificates extends viewModelBase {
...
+    canBeAutomaticallyRenewed(thumbprints: string[]) {
         return ko.pureComputed(() => {
             return _.includes(thumbprints, this.serverCertificateThumbprint()) && this.serverCertificateSetupMode() === 'LetsEncrypt';
         });
     }
     
     renewServerCertificate() {
-        return new forceRenewServerCertificateCommand()
-            .execute();
+        this.confirmationMessage("Server certificate renewal", "Do you want to renew server certificate?", ["No", "Yes, renew"])
```
Comment (iftahbe) [inline src/Raven.Studio/typescript/viewmodels/manage/certificates.ts]: Do you want to renew the server certificate?

Comment (ml054) [reply src/Raven.Studio/typescript/viewmodels/manage/certificates.ts]: What do we renew exactly ?

Comment (iftahbe) [reply src/Raven.Studio/typescript/viewmodels/manage/certificates.ts]: I meant, fix the grammar, add a 'the'.

Comment (ml054) [reply src/Raven.Studio/typescript/viewmodels/manage/certificates.ts]: done - thanks :)

**Pull request merged**